### PR TITLE
feat(cloudflare): support astro:env

### PR DESCRIPTION
**Pull request by alexanderniebuhr**
## Changes

- Adds support for `getSecret`

## Testing

Integration test

## Docs

N/A, I think we need to add some precisions in the config reference since it's scoped to the request

### CODE REVIEW

**Comment by alexanderniebuhr** [comment]
cc @florian-lefebvre

**Comment by florian-lefebvre** [comment]
Thanks I'll have a look!

File: packages/cloudflare/src/entrypoints/image-service.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ const service: ExternalImageService = {
 		}
 
 		const imageEndpoint = joinPaths(
+			// @ts-expect-error - Property 'env' does not exist on type 'ImportMeta'.ts(2339)
```
Comment (alexanderniebuhr) [inline packages/cloudflare/src/entrypoints/image-service.ts]: I thought this was fixed with the update to the matching/newest vite version 🤔

Comment (florian-lefebvre) [reply packages/cloudflare/src/entrypoints/image-service.ts]: No this was by fixed by the `astro/client` reference

File: packages/cloudflare/src/utils/non-server-chunk-detector.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import type { OutputBundle } from 'rollup';
-import type { Plugin } from 'vite';
+import type { Plugin, PluginOption } from 'vite';
```
Comment (alexanderniebuhr) [inline packages/cloudflare/src/utils/non-server-chunk-detector.ts:2]:
```suggestion
import type { PluginOption } from 'vite';
```

**Pull request merged**